### PR TITLE
Ties modal lip width to right rail width

### PR DESCRIPTION
**Pull request by abea**
Fixes the funky right rail lip. Width didn't match the right rail. https://apostrophecms.atlassian.net/browse/A30U-438

💋

### CODE REVIEW

**Review by gregvanbrug — APPROVED** [review]

File: modules/@apostrophecms/modal/ui/apos/components/AposModal.vue [hunk 1]
```diff
@@ -383,14 +383,14 @@ export default {
   }
 
   .apos-modal__main--with-rails {
-    grid-template-columns: 18% 1fr minmax(250px, 22%);
+    grid-template-columns: 18% 1fr minmax(250px, $modal-rail-right-w);
```
Comment (falkodev) [inline modules/@apostrophecms/modal/ui/apos/components/AposModal.vue:386]: I have this when visualizing an image:
image

The button "published/unpublished" is is wider than the rail (might depend on the window's width). Only the "Save" button is present (and is not doing anything, but that might be done in another PR I guess).

Comment (abea) [reply modules/@apostrophecms/modal/ui/apos/components/AposModal.vue:386]: That is true. Yes, this stuff would be separate issues. The save button should work in the 3.0 branch right now.

**Review by stuartromanek — APPROVED** [review]
